packages/unified-latex-to-pretext/libs/author-info.ts

```diff
@@ -0,0 +1,66 @@
+import * as Ast from "@unified-latex/unified-latex-types";
+import { visit } from "@unified-latex/unified-latex-util-visit";
+import { match } from "@unified-latex/unified-latex-util-match";
+import { htmlLike } from "@unified-latex/unified-latex-util-html-like";
+import { printRaw } from "@unified-latex/unified-latex-util-print-raw";
+
+/**
+ *
+ * Visits all the matching nodes and gathers author information, then send them to render and output pretext.
+ */
+export function gatherAuthorInfo(ast: Ast.Ast): any[] {
```

Comment:
We will need to change the return type because we also need to output a list of warnings. These warnings should be stored in a VFileMessage.